SimG4CMS/Calo/src/CaloSD.cc

```diff
@@ -28,7 +28,7 @@
 #include <memory>
 #include <sstream>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
@bsunanda , please, comment out the DEBUG

Reply:
Done